src/ItemLinkManagement/Public/New-ItemLink.Tests.ps1

```diff
@@ -138,27 +138,27 @@ Describe "New-ItemLink" -Tag 'Unit' {
             }
         }
 
-        # It 'Should create a Junction for directory if it already exists when using -Force (Windows)' {
-        #     if ($env:OS -eq 'Windows_NT') {
-        #         $testDirectory = New-Item $testDirectoryFullName -ItemType Directory
-        #         $testDirectoryLink = New-Item $testDirectoryLinkFullName -Value $testDirectory.FullName -ItemType 'Junction'
-        #         $params = @{
-        #             ItemType = 'Junction'
-        #             Path = $testDirectoryLink.FullName
-        #             Value = $testDirectory.FullName
-        #             Force = $true
-        #             ErrorAction = 'Stop'
-        #         }
-        #         New-Item @params > $null
-
-        #         $result = New-ItemLink @params
-
-        #         $result | Should -BeOfType [System.IO.DirectoryInfo]
-        #         $result.LinkType | Should -Be $params['ItemType']
-        #     }else {
-        #         $true
-        #     }
-        # }
+        It 'Should create a Junction for directory if it already exists when using -Force (Windows)' {
+            if ($env:OS -eq 'Windows_NT') {
+                $testDirectory = New-Item $testDirectoryFullName -ItemType Directory
+                $testDirectoryLink = New-Item $testDirectoryLinkFullName -Value $testDirectory.FullName -ItemType 'Junction'
+                $params = @{
+                    ItemType = 'Junction'
+                    Path = $testDirectoryLink.FullName
+                    Value = $testDirectory.FullName
+                    Force = $true
+                    ErrorAction = 'Stop'
+                }
+                New-Item @params > $null
```

Comment:
@joeltimothyoh this line should be removed

Reply:
Yes but locally it still doesn’t work. Still figuring out cuz i’m getting the identical stack locally @leojonathanoh

Reply:
@joeltimothyoh this is a line that should not exist

Reply:
I know. I was debugging earlier today but that isn’t the cause @leojonathanoh